angular/src/app/modules/mytimesheet/mytimesheet-tardiness/mytimesheet-tardiness.component.ts

```diff
@@ -98,6 +98,14 @@ export class MytimesheetTardinessComponent extends AppComponentBase implements O
     return "green";
   }
 
+  getCss2(value) {
```

Comment:
1. Change the function name to clearly by purpose. Don't use the general name.
2. Add try-catch block to this function to avoid exception when parse value